documentation/healthcare-analytics-workspace.adoc

```diff
@@ -0,0 +1,471 @@
+== Healthcare Analytics
+:images: {img}
+:data: https://neo4j-graph-examples.github.io/healthcare-analytics/data
+
+image::{img}/healthcare-analytics.jpg[float=right,width=200px]
+
+Health care analytics allows for the examination of patterns in healthcare data to determine how clinical care can be improved. 
+
+Healthcare organizations can _"realize new opportunities and efficiencies by leveraging the connections within their existing data: be it in a connected genome, or a provider network, or patient treatments,"_.
+*Emil Eifrem, Neo4j CEO*
+
+Graph databases are a good fit for healthcare analytics due to their ability to model and query highly connected data.
+
+In this guide, you will learn:
+
+* ...
```

Comment:
TBD?

Reply:
To be defined. Its a placeholder for me to complete once I know what they are going to learn. Thanks